Changes

```diff
@@ -64,6 +64,16 @@ Working version
   (Enguerrand Decorne, report by Jon Ludlam,
   review by Tom Kelly, KC Sivaramakrishnan and Gabriel Scherer)
 
+- #11082 The runtime now reserves only minor_heap_wsz words of
+  memory per domain for the minor heap area (down from 2GB per domain).
+  Calling Gc.set asking for a larger minor heap than the current
+  reservation will un-reserve and re-reserve the minor heap area
+  of all domains. Gc.set changes the minor heap size only for
+  the current domain.
+  Tools like AFL or Valgrind can now be used on compiled programs.
```

Comment:
Again, AFL and Valgrind were somehow usable since commit aff58350d.  What did not work is programs like Coq and CompCert that request big minor heaps.